Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby File.read(".ruby-version").chomp
+ruby '2.5.3'
```

Comment:
why this change? It's still better to set the ruby `.ruby-version` as this is used by Rbenv...

Reply:
Docker couldn't read the file. Maybe theres another way to do it...